pkg/client/mocks/Factory.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.30.1. DO NOT EDIT.
+// Code generated by mockery v2.35.4. DO NOT EDIT.
```

Comment:
since you did not change `go.mod`, should this file been changed?

Reply:
@mateusoliveira43 mockery isn't in the built image (which go.mod reflects) -- mockery is a tool used to generate golang code for mocks.

Reply:
This comment is inserted by mockery to indicate what version generated the code.

Reply:
Got it Scott, thanks! We can resolve this conversation then

Just curious: where is mockery in velero repo?

Reply:
It's not in the repo directly, but it's referenced in docs: https://github.com/vmware-tanzu/velero/blob/main/site/content/docs/main/code-standards.md#mocks
example from the docs:
```
go get github.com/vektra/mockery/.../
cd pkg/podvolume
mockery -name=Restorer
```